assets/blocks/quiz/quiz-block/pagination-settings.js

```diff
@@ -172,51 +155,18 @@ export const PaginationSidebarSettings = ( { settings, onChange } ) => {
 					</>
 				) }
 			</PanelBody>
-			<PanelColorSettings
-				title={ __( 'Color settings', 'sensei-lms' ) }
-				initialOpen={ false }
-				colorSettings={ [
-					{
-						value: progressBarColor,
-						onChange: ( value ) =>
-							onChange( {
-								...settings,
-								progressBarColor: value,
-							} ),
-						label: __( 'Progress bar color', 'sensei-lms' ),
-					},
-					{
-						value: progressBarBackground,
-						onChange: ( value ) =>
-							onChange( {
-								...settings,
-								progressBarBackground: value,
-							} ),
-						label: __(
-							'Progress bar background color',
-							'sensei-lms'
-						),
-					},
-				] }
-			>
-				<ContrastChecker
```

Comment:
Just curious if this was removed intentionally?

Reply:
I've decided to remove it because to make it work, two of these components need to be added (for both the button and progress bar colors) and that looks a bit weird when both are displayed:

<img width="276" alt="Screenshot on 2022-01-14 at 22-34-56" src="https://user-images.githubusercontent.com/1612178/149581705-09005cab-23fe-4523-898e-fa424783b611.png">
 
Thinking about it now, it doesn't look that bad. Should I include them?

Reply:
Since it's impossible to tell which color combo the notice refers to, sounds like it was a good call to take it out.